database/seeds/MagazineTableSeeder.php

```diff
@@ -1,24 +0,0 @@
-<?php
-
-use Carbon\Carbon;
-use Illuminate\Database\Seeder;
-
-class MagazineTableSeeder extends Seeder
-{
-    /**
-     * Run the database seeds for Magazine table.
-     *
-     * @return void
-     */
-    public function run()
-    {
-        DB::table('magazines')->insert(
-            [
-                'name' => "August 2016",
-                'short_name' => 'Aug16',
-                'no_of_articles' => 0,
-                'created_at' => Carbon::now()->format('Y-m-d H:i:s')
-            ]
-        );
-    }
-}
```

Comment:
These changes are not related to login form.

This probably happened due to wrong branch creation/merges.